account/forms.py

```diff
@@ -14,6 +14,9 @@
 
 alnum_re = re.compile(r"^\w+$")
 
+User = get_user_model()
+USER_FIELD_MAX_LENGTH = User._meta.get_field(User.USERNAME_FIELD).max_length
```

Comment:
```suggestion
USER_FIELD_MAX_LENGTH = getatter(User, User.USERNAME_FIELD).field.max_length
```

Reply:
Your suggestion won't work - you misspelled `getattr` (added an extra `e` in `attr`).